app/api/attendees.py

```diff
@@ -1,7 +1,7 @@
 from datetime import datetime
 
 from flask import Blueprint, request, jsonify, abort, make_response
-from flask_jwt import current_identity, jwt_required
+from flask_jwt import current_identity
```

Comment:
Why did you change this?

Reply:
@kushthedude Was not used anywhere in file, so I guess Pycharm did it.